olive/packaging/packaging_generator.py

```diff
@@ -73,3 +75,37 @@ def _package_candidate_models(tempdir, footprint: Footprint, pf_footprint: Footp
         metric_path = str(model_dir / "metrics.json")
         with open(metric_path, "w") as f:
             json.dump(node.metrics.value, f)
+
+
+def _download_onnxruntime_package(cur_path):
+    try:
+        import onnxruntime
+    except ImportError:
+        logger.warning("onnxruntime is not installed, skip packaging onnxruntime package")
+        return
+
+    ort_version = onnxruntime.__version__
+
+    try:
+        # Download Python onnxruntime package
+        python_downlaod_path = str(cur_path / "sample_code" / "ONNXModel" / "python" / "ONNXRuntime")
+        downlaod_command = f"python -m pip download onnxruntime=={ort_version} --no-deps -d {python_downlaod_path}"
+        run_subprocess(downlaod_command)
```

Comment:
There two separate packages. onnxruntime and onnxruntime-gpu.

onnxruntime-gpu should be downloaded when it is recommended.

Reply:
Added.